pkg/helm/helm.go

```diff
@@ -15,17 +15,12 @@
 package helm
 
 import (
-	"bytes"
-	"compress/gzip"
-	"encoding/base64"
-	"encoding/json"
 	"fmt"
-	"io/ioutil"
-	"regexp"
-	"strconv"
 
-	v1 "k8s.io/api/core/v1"
-	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	helmstorage "helm.sh/helm/pkg/storage"
+	"helm.sh/helm/pkg/storage/driver"
```

Comment:
Just noticed this. I'll change the names to be helmstorage2 and driver2 for clarity.